unified-codes/libs/data/src/lib/types/Entity.ts

```diff
@@ -1,9 +1,22 @@
 import { Property } from './Property';
 
+export enum EEntityType {
+  DRUG = 'medicinal_product',
+  UNIT_OF_USE = 'unit_of_use',
```

Comment:
Hm, disagree that this isn't required, so will give my motivation for your thoughts:

When filtering/searching etc. this gives us compile time type checking that could otherwise cause bugs at runtime. e.g. should ensure that the toggle bar only can toggle types that are valid, otherwise we are passing around strings which could mean anything, and typos such as `medical_product` versus `medicinal_product` could cause hard to detect bugs etc. Having an enum of valid types means that we also have a reference for all the types which is shared by the front and back-end, so ensures they are speaking the same language (i.e. front-end isn't trying to filter by a type that the back-end doesn't know).

Thoughts?

Reply:
yep, fair enough. I'm thinking too specifically for the current usage.